include/mbedtls/pk_internal.h

```diff
@@ -1,10 +1,12 @@
 /**
  * \file pk_internal.h
  *
- * \brief Public Key abstraction layer: wrapper functions
- */
-/*
- *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+ * \brief Public Key cryptography abstraction layer: built-in key types
```

Comment:
As noted by Manuel, we should clarify who should read this file.

> `\brief` Public-key cryptography abstraction layer: internal definitions

> This file contains built-in types for handling natively supported key types using the interface defined in `pk_info.h`.

> `\warning` This file contains internal definitions for the library. The interfaces in this file may change in future versions of the library without notice.

Reply:
Done